README.md

```diff
@@ -1,2 +1,60 @@
 # bard-client
 JavaScript library for shared client-side analytics across DSP
+
+# Overview
+
+This library is designed to be imported by custom analytics code in Data
+Sciences Platform (DSP) web UIs like Terra and Single Cell Portal.  It supplies
+them with properties common across DSP, like "referrer", "browser",
+"screen width", and more.
```

Comment:
I might add a comment here along the lines of `eventually, this library may be expanded to also include both a utility 'logToBard' method, and or default instrumentation of events, but for now its focus is on standardizing a set of event properties.`

Reply:
Agreed, done in 0c81ec6.